lib/msal-browser/src/crypto/BrowserCrypto.ts

```diff
@@ -63,6 +92,20 @@ export class BrowserCrypto {
         return this.hasIECrypto() || this.hasBrowserCrypto();
     }
 
+    /**
+     * Checks whether IE crypto (AKA msCrypto) is available.
+     */
+    private hasIECrypto(): boolean {
+        return "msCrypto" in window;
```

Comment:
Looks like "msCrypto" is used in multiple places, could probably be made into a constant.